File: 08-27-avl/EjemploIo.java
import java.util.Scanner;

public class EjemploIo {
  public static void main(String[] args) {
    String nombre;
    int edad;

    Scanner sc = new Scanner(System.in);
    System.out.println("Como es tu nombre?");
    nombre = sc.next();
    System.out.println("Cual es tu edad?");
    edad = sc.nextInt();

    System.out.println("Hola " + nombre + ", tu edad es " + edad);
  }
}

File: 08-27-avl/ejemplo_io.cpp
#include <iostream>
#include <string>

int main()
{
  std::string nombre;
  int edad;

  std::cout << "Como es tu nombre?" << std::endl;
  std::cin >> nombre;

  std::cout << "Cual es tu edad?" << std::endl;
  std::cin >> edad;

  std::cout << "Hola " << nombre << ", tu edad es " << edad << std::endl;
}
